packages/adblocker/src/filters/cosmetic.ts

```diff
@@ -396,7 +402,7 @@ export default class CosmeticFilter implements IFilter {
    * symetrical to the one in `serializeCosmeticFilter`.
    */
   public static deserialize(buffer: StaticDataView): CosmeticFilter {
-    const mask = buffer.getUint8();
+    const mask = buffer.getUint16();
```

Comment:
Just leaving an idea. Maybe it's not good, but we may merge this with `optionalParts`.